packages/zowe-explorer/__tests__/__unit__/utils/LoggerUtils.unit.test.ts

```diff
@@ -1,3 +1,14 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the *
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+ * https://www.eclipse.org/legal/epl-v20.html                                      *
+ *                                                                                 *
+ * SPDX-License-Identifier: EPL-2.0                                                *
+ *                                                                                 *
+ * Copyright Contributors to the Zowe Project.                                     *
```

Comment:
Missing license header?
This is a test.ts file, so no biggie.
However, should we run all our files through a license-script?

Thanks for adding it! :yum: